charts/pxc-operator/templates/deployment.yaml

```diff
@@ -18,6 +18,10 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+    {{- with .Values.podAnnotations }}
```

Comment:
Any reason not to call it just `.Values.annotations`? Looking at our charts only pmm in one place has podAnnotations, all others seem to use just `annotations`

Reply:
well podAnnotations is quite standard and a good name for this